Automattic/jetpack
# Update the README to include information about Actions::do_only_first_initial_sync

#### Changes proposed in this Pull Request:
* Update the Sync package readme to include information about Actions::do_only_first_initial_sync

#### Jetpack product discussion
* n/a

#### Does this pull request change what data or activity we track or use?
* no

#### Testing instructions:
Verify that the changes to the Sync package README file make sense and are useful.
